walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,13 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String cleanName="";
+    for(int i=0;i<name.length();i++){
+      Character c= name.charAt(i);
```

Comment:
Run formatter, spacing is inconsistent. Trying formatting in VSCode or I think Kwame got prettier working for Java files.